coral/src/app/layout/header/ProfileDropdown.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Box,
+  DropdownMenu,
+  Typography,
+  Icon,
+  useToast,
+} from "@aivenio/aquarium";
+import user from "@aivenio/aquarium/dist/src/icons/user";
+import logOut from "@aivenio/aquarium/dist/src/icons/logOut";
+import classes from "src/app/layout/header/ProfileDropdown.module.css";
+import { logoutUser } from "src/domain/auth-user";
+
+type MenuItem = {
+  path: string;
+  name: string;
+};
+
+const menuItems: MenuItem[] = [
+  {
+    path: "/myProfile",
+    name: "My profile",
+  },
+  {
+    path: "/tenantInfo",
+    name: "My tenant info",
+  },
+  {
+    path: "/changePwd",
+    name: "Change password",
+  },
+];
+
+const LOGOUT_KEY = "logout";
+function ProfileDropdown() {
+  const toast = useToast();
+  function navigateToAngular(path: string) {
+    window.location.assign(`${window.origin}${path}`);
+  }
+
+  function onDropdownClick(actionKey: string | number) {
+    if (actionKey === LOGOUT_KEY) {
+      toast({
```

Comment:
I've not found a reasonable way to remove a toast notification (asked in DS channel, maybe that can be added in the future) other then e.g. passing a key and through the parent component force a re-mounting, but I decided that's way to much effort for an case that (hopefully) will not really happen (it never happened to me in the last year :D)

Worst case is that there is an error from the server and the user sees both notifications. Since the error is RED etc. I think it will be clear that there is an error, so it's not confusing, only a bit wonky. 

I set the time to 30 seconds, so in case the request takes long due to bad connection, the notification does not disappear before the endpoint answered.

Reply:
I think it's pretty ok like this ^^